org.OpenT2T.Onboarding.LightifyHub/node/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "opent2t-onboarding-lightifyhub",
+    "version": "1.0.1",
```

Comment:
Convention is to start at 0.1 for something that is still under development, not yet "released".